monitor/palma_analyze

```diff
@@ -96,8 +96,39 @@ class PalmaLogfile(object):
               (os.path.basename(self.filename), len(self.data),
                totUsers, totLength)
         )
-
+    
+    def printMonthlySummary(self):
+          
+        def print_bin(month_bin):
+            length = datetime.timedelta()
+            users = 0
+            for x in month_bin: 
+                length += x.end - x.start
+                users += x.nuser
+            sessions = len(month_bin)
+            avg_length = length // sessions
```

Comment:
Wow, I was not aware of the `//` operator! Is there a specific reason you are using it here instead of `/` in the context of `timedelta`? It does not seem to make any difference, at least at the precision of seconds.

Reply:
I was testing out the division operators to see if the results would be different, but they are not in this case with timedelta.

Reply:
```suggestion
            avg_length = length / sessions
```
Ok, so maybe we can just use the "normal" division operator here.